docs/src/pages/components/button.mdx

```diff
@@ -25,17 +43,21 @@ rule, each page should have one primary button. Any remaining calls-to-action ar
 </Component>
 ```
 
-#### Outline Button
-```tsx live
-<Component initialState={{ size: 'md' }}>
+<br />
+
+### Variants
```

Comment:
I think I would still include the `Filled` variant here so that the three options are clearly enumerated visually (even if redundant to the one above).

I also feel like it might be good to keep the variant buttons looking "default" until clicked (where it can pick up whatever custom values) so that the differences between the variants are more clear.

That or the three variants could be combined into a single live coding example, so as to be seen side by side.